src/Styleguide/Pages/Artwork/Components/__stories__/ArtistInfo.story.tsx

```diff
@@ -13,7 +13,7 @@ storiesOf("Styleguide/Artwork", module).add("ArtistInfo", () => {
       <Section title="Artist Info">
         <Grid>
           <Row>
-            <Col xl={8} lg={8} md={8} sm={12} xs={12}>
+            <Col md={8}>
```

Comment:
@zephraph its best to go from smallest to largest when defining col breakpoints. Also a couple other things:
- if a given breakpoint is set it will automatically apply to every larger breakpoint
- default breakpoint col is 12; since `xs` and `sm` = 12 they can be omitted.